lib/widgets/content.dart

```diff
@@ -107,6 +109,27 @@ class BlockContentList extends StatelessWidget {
   }
 }
 
+/// A Divider matching the style found in:
+/// [web/styles/rendered_markdown.css](https://github.com/zulip/zulip/blob/main/web/styles/rendered_markdown.css)
+class ThematicBreak extends StatelessWidget {
+  const ThematicBreak({
+    super.key,
+    this.thickness = 2,
+    this.color,
+  });
```

Comment:
Let's remove these params and just hard-code the intended values. This element doesn't need to appear with different thicknesses or colors in different places.